jena-sdb/assembly.xml

```diff
@@ -41,11 +41,11 @@
 
   <files>
     <file>
-      <source>dist/LICENSE</source>
+      <source>LICENSE</source>
```

Comment:
I changed this because the Maven assembler complained that "dist/LICENSE" didn't exist. I understood these paths to be relative to the jena-sdb directory.

Maybe change to "../dist/LICENSE"?